### PR TITLE
INS-384: Add method for getting jet ref from record ref.

### PR DESCRIPTION
**Pull request by andreyromancev**
Closes INS-384

### CODE REVIEW

File: ledger/jetcoordinator/jetnode.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+type JetNode struct {
+	threshold uint64
+	ref       *core.RecordRef
+	left      *JetNode
+	right     *JetNode
+}
+
+func (jn *JetNode) GetContaining(objRef uint64) *core.RecordRef {
```
Comment (ita-sammann) [inline ledger/jetcoordinator/jetnode.go]: Shall it be 64 bytes, not bits?

File: ledger/jetcoordinator/jetnode.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	}
+
+	switch bytes.Compare(objRef[:], jn.ref[:]) {
+	case 0:
+		return &jn.ref
+	case -1:
+		return jn.left.GetContaining(objRef)
+	default:
+		return jn.right.GetContaining(objRef)
+	}
```
Comment (ita-sammann) [inline ledger/jetcoordinator/jetnode.go]: Why 3 cases? Only leafs of tree should represent Jets

Comment (andreyromancev) [reply ledger/jetcoordinator/jetnode.go]: case 0: its jet itself
The rest two are for branches.

File: ledger/jetcoordinator/jetcoordinator.go [hunk 3]
```diff
@@ -107,7 +109,23 @@ func (jc *JetCoordinator) getNextValidators(candidates [][]byte, count int) ([][
...
 
+func (jc *JetCoordinator) jetRef(objRef core.RecordRef) *core.RecordRef { // nolint: megacheck
+	return jc.rootJetNode.GetContaining(&objRef)
+}
+
 // NewJetCoordinator creates new coordinator instance.
 func NewJetCoordinator(db *storage.DB) (*JetCoordinator, error) {
-	return &JetCoordinator{db: db}, nil
+	// TODO: temporary jet tree (later will be fetched fom config or ledger)
+	rootJetNode := &JetNode{
```
Comment (ita-sammann) [inline ledger/jetcoordinator/jetcoordinator.go]: Same thing as below. Not all tree nodes are Jets

Comment (andreyromancev) [reply ledger/jetcoordinator/jetcoordinator.go]: They used here as thresholds. If node doesn't have children it leaf, then we use it as ref. In other cases we treat it like splitting value.

**Pull request merged**